app/src/widgets/dishcard.tsx

```diff
@@ -1,17 +1,40 @@
 import '../styles/index.css'
+import {useEffect, useState } from "react";
+import axios from "axios";
+import mug from '../assets/mug_icon_contained.svg'
+import container from '../assets/dish_icon_contained.svg'
 
-export default ({dish}) => {
-    const dateFormat = { day: 'numeric', year: 'numeric', month: 'short'}
+export default ({dish,token}) => {
+  const options = { weekday: 'long', year: 'numeric', month: 'long'};
+  const [dishAPI, setDishAPI] = useState([])     
+
+    useEffect(()=>{
+        axios.get('http://ec2-34-213-210-231.us-west-2.compute.amazonaws.com/api/dish', {headers:{"x-api-key":"test","session-token":token}, params:{"id":dish.dish}})
+        .then(function (response) {
+          setDishAPI(response.data.dish)
+        })
+        .catch(function (error) {
+          console.log(error);
+        });
+      },[])
+
+    if (dishAPI['type'] == 'mug'){
+      var icon = mug
```

Comment:
please don't use `var`, it's outdated! you can make this one line with a ternary

`const icon = dishAPI.type === mug ? mug : container`